app/models/util/table_exporter.rb

```diff
@@ -2,9 +2,12 @@ module Util
   class TableExporter
     attr_reader :zipfile_name, :table_names
 
-    def initialize(tables=[])
-      @temp_dir     = "#{Util::FileManager.new.dump_directory}/export"
-      @zipfile_name = "#{@temp_dir}/#{Time.zone.now.strftime('%Y%m%d')}_export.zip"
+    def initialize(tables=[],schema='')
+      @schema = schema
+      @temp_dir     = "#{Util::FileManager.new.dump_directory}/export" unless schema == 'beta'
+      @temp_dir     = "#{Util::FileManager.new.dump_directory}/export_beta" if schema == 'beta'
+      @zipfile_name = "#{@temp_dir}/#{Time.zone.now.strftime('%Y%m%d')}_export.zip" unless schema == 'beta'
+      @zipfile_name = "#{@temp_dir}/#{Time.zone.now.strftime('%Y%m%d')}_beta_export.zip" if schema == 'beta'
```

Comment:
Might be a little more readable if we have an `if else` structure. For example:

```ruby
if schema == 'beta'
  @temp_dir = "..."
  @zipfile_name = "..."
else
  @temp_dir = "..."
  @zipfile_name = "..."
end
```

Having multiple lines in the if part makes it so you don't have to cognitively parse the if statement for each line